lib/presentation/ui/_base/base_page.dart

```diff
@@ -23,8 +23,10 @@ abstract class BaseState<T extends BasePage> extends State<T> {
   @override
   @mustCallSuper
   void initState() {
+    super.initState();
     widget.bloc.errorStream
-        .transform(new ThrottleStreamTransformer(new Duration(seconds: 2)))
+        .transform(new ThrottleStreamTransformer(
```

Comment:
What are you actually trying to achieve here?

Reply:
That is from an old commit ahaha

Well, if an error is fired twice in a rapid succession (2 API calls fail at the same time, they will add an event to the error stream), the Snackbar will be shown multiple times.

If we do it like this then we are throttling the events with the approximate duration of a Snackbar LONG